src/main.rs

```diff
@@ -76,6 +77,7 @@ async fn axum(#[shuttle_shared_db::Postgres] pool: PgPool) -> shuttle_axum::Shut
         .route_layer(login_required!(Backend, login_url = "/auth"))
         .merge(hypermedia::router::auth::router())
         .merge(hypermedia::router::validation::router())
+        .layer(HelmetLayer::new(Helmet::default().add(XFrameOptions::Deny)))
```

Comment:
![image](https://github.com/nicolasauler/finnish/assets/63314440/d9dc4a8b-e345-48f6-ae48-dab9d30ee4ca)
App broken due to CSP in htmx extensions

Reply:
Currently htmx.min.js breaks as having an inline script inside it

Reply:
Check if fix in https://github.com/nicolasauler/finnish/pull/28/commits/08b75607405def12f81745b8830d5805454bb500

Reply:
Wasn't fixed there, but was in https://github.com/nicolasauler/finnish/pull/28/commits/4291ca522a7ac82da4f408700e13081f52bdaf09
Note that this fix broke some functionality of CSP in the first place, so a better solution needs to be found

Reply:
Opened issue at #29